data/sles4sap/qe_sap_deployment/qesap_azure_uri.yaml

```diff
@@ -1,35 +1,41 @@
-provider: "%PROVIDER%"
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+# Maintainer: QE-SAP <qe-sap@suse.de>
+# Summary: Generic yaml template for use with qe-sap-deployment project: https://github.com/SUSE/qe-sap-deployment
+provider: 'azure'
 apiver: 3
 terraform:
   variables:
-    az_region: "%REGION%"
-    deployment_name: "%DEPLOYMENTNAME%"
-    public_key: "%SSH_KEY_PUB%"
-    hana_remote_python: "%ANSIBLE_REMOTE_PYTHON%"
-    iscsi_remote_python: "%ANSIBLE_REMOTE_PYTHON%"
+    az_region: '%REGION%'
+    deployment_name: '%DEPLOYMENTNAME%'
+    public_key: '%SSH_KEY_PUB%'
+    hana_remote_python: '%ANSIBLE_REMOTE_PYTHON%'
+    iscsi_remote_python: '%ANSIBLE_REMOTE_PYTHON%'
     os_image_uri: 'https://%STORAGE_ACCOUNT_NAME%.blob.core.windows.net/sle-images/%OS_VER%'
-    hana_instancetype: "%HANA_INSTANCE_TYPE%"
+    hana_instancetype: '%HANA_INSTANCE_TYPE%'
```

Comment:
`qe-sap-deployment/terraform/azure` does not have variable like `hana_instancetype`. It is an AWS only variable.